lz4-sys/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "lz4-sys"
+license = "MIT"
+links = "lz4"
+version = "1.19.173"
```

Comment:
You could chose to have a version number here that's independent from the sys crate.